src/assets/config/config.js

```diff
@@ -0,0 +1,10 @@
+
+// See ServicesConfig for format
+window.showcaseConfig = {
+  "backend": {
+    "serverUrl": "http://localhost:4000/graphql",
+    "wsServerUrl": "ws://localhost:4000/graphql"
+  },
+  // "push": {},
+  // "auth": {}
```

Comment:
I think we should put an example of push and auth config or at least a link to keycloak and push documentation

Reply:
Initially, I tried to use this configuration: 
![image](https://user-images.githubusercontent.com/7964685/69962694-d118a480-150e-11ea-9036-a02a83e51f84.png)
but the url name was wrong, so I had to manually go in the code to find out which one was  the correct format